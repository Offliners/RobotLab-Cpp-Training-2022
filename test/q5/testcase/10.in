9/18/14-6/(8-10)=*17+(9/16-6*17-12-10+8/(19-8+10/14))=-14+13+19+7*15*5-17*12/9-1*13/13*19+((19-8)-17)/17*18/17-(9+3)*(1-(7*6+9))%14=+20/12/(7-13)=
