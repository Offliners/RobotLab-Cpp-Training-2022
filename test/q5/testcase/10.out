-58
-159
-334
-686
-707
